docs/scripts/cibuild.sh

```diff
@@ -2,4 +2,4 @@
 set -e # halt script on error
```

Comment:
Just delete this file. It's not used anymore; the old travis build used it on the old pages repo; now everything is just in the action yml file.